packages/ui/src/components/Chip/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta } from '@storybook/react'
+import { Chip } from '..'
+
+export default {
+  component: Chip,
+  decorators: [
+    StoryComponent => (
+      <div style={{ display: 'flex', flexDirection: 'column' }}>
+        <StoryComponent />
+      </div>
+    ),
+  ],
+  title: 'Components/Badges/Chip',
+} as Meta<typeof Chip>
+
+export { Playground } from './Playground.stories'
+export { Icons } from './Icons.stories'
+export { Groups } from './Groups.stories.tsx'
+export { Size } from './Size.stories'
```

Comment:
Would be nice also to have an example with disabled state. With and without the Chip.Icon having a onclick on it so we can check if it works normally on this state.